devel/contents.tex

```diff
@@ -48,10 +48,16 @@ \section{Examination}%
 The report will be graded according to the following grading criteria:
 \begin{description}
   \item[Grade E] You fulfil all the \acp{ILO} at the minimum level.
+
+  \item[Grade C] You fulfil all the \acp{ILO},
```

Comment:
What is the difference of fulfiling all the ILO, and all the ILO at minimum level? If you fulfill the ILO you fulfill the ILO?

Reply:
Yes, they are the same. It's just that the first emphasises that it's the minimum you can do.